sdk-cpp/CMakeLists.txt

```diff
@@ -79,11 +69,52 @@ if(${DO_PLATFORM_ID} EQUAL ${DO_PLATFORM_ID_LINUX})
         DOSVC_BIN_NAME="${DOSVC_BIN_NAME}"
         DO_PLUGIN_APT_BIN_NAME="${DO_PLUGIN_APT_BIN_NAME}"
     )
-    target_link_libraries(${DO_SDK_LIB_NAME}
-        PRIVATE doversion cpprestsdk::cpprest
-        PUBLIC ${Boost_LIBRARIES}
+
+    set(sdk_public_includes
+        $<BUILD_INTERFACE:${${PROJECT_NAME}_SOURCE_DIR}/include>
+        $<INSTALL_INTERFACE:include/${DO_SDK_LIB_NAME}>
+    )
+    set(sdk_private_includes
+        "src/internal/rest"
+        "src/internal/rest/util"
+        "src/internal"
+        ${docs_common_includes}
```

Comment:
>docs_common_includes [](start = 10, length = 20)

${docs_common_includes} seems to be not used anymore, right? Even before this PR I mean.

Reply:
yeah removed without issue

---
In reply to: [606372614](https://github.com/microsoft/do-client/pull/19#discussion_r606372614) [](ancestors = 606372614)